sass/_SimpleFilter.scss

```diff
@@ -0,0 +1,34 @@
+@import "bourbon/bourbon";
+@import "Variables";
+
+.CoveoSimpleFilter {
```

Comment:
aim for "flat" rules.

There's should be as less as possible "nested" rules. TO do that, use longer class name to ensure specificity.